build.sbt

```diff
@@ -1,6 +1,10 @@
 lazy val root = project.in(file("."))
+  .aggregate(`scala-library-next`.jvm, `scala-library-next`.js)
   .settings(
-    publish / skip := true
+    publish / skip := true,
+    // With CrossType.Pure, the root project also picks up the sources in `src`
+    Compile / sources := Nil,
+    Test / sources := Nil,
```

Comment:
Consider emptying out `unmanagedSourceDirectories` instead of`sources`. The former will be understood by IDEs in addition to sbt compilation.